clang/test/Driver/sycl-offload-tempfile.cpp

```diff
@@ -0,0 +1,36 @@
+// REQUIRES: x86-registered-target
```

Comment:
Why this test requires x86 target?

Reply:
Hmm - shouldn't really.  It's generic code and we are only testing the tempfile removal behavior.  It may be there from when I cloned the source from an existing test and refactored to fit my needs.  I'll clean it out.